.coafile

```diff
@@ -0,0 +1,27 @@
+[all]
+ignore = node_modules/**, bower_components/**, dist/**, .eslintrc.js, tmp/**,
+         .git/**
+
+[all.SpaceConsistencyBear]
+files = **.js, **.json, **.yml, **.md, .eslintrc
+bears = SpaceConsistencyBear
+use_spaces = true
+
+[all.json]
+bears = JSONFormatBear
+files = **.json
+indent_size = 2
+
+[all.js]
+bears = ESLintBear
+eslint_config = .eslintrc.js
+files = *.js, app/**.js, config/*.js, tests/**.js
```

Comment:
Ideally `**.js` , and using ignore to handle any files which are special cases and should not be linted by this bear.

Reply:
Hanged in my machine. Not a big deal, Use `**.js`

Reply:
https://travis-ci.org/coala/git-task-list/jobs/391031526